pineappl_py/src/grid.rs

```diff
@@ -326,8 +342,9 @@ impl PyGrid {
     ///
     /// Parameters
     /// ----------
-    /// slices : Iterable
-    ///     list of (PyOperatorSliceInfo, 5D array) describing each convolution
+    /// slices : list(list(tuple(PyOperatorSliceInfo, PyReadOnlyArray4)))
+    ///     list of EKOs where each element is a list of (PyOperatorSliceInfo, 5D array)
```

Comment:
```suggestion
    ///     list of EKOs where each element is a list of (PyOperatorSliceInfo, 4D array)
```
?

Reply:
Yes, it is indeed a 4D,

Btw, this is what I briefly asked in yesterday's CM re `pineko` (to which I now know the answer). For the time being, each operator is passed as a vector but it'll be changed to a generator.